omero/training-server/playbook.yml

```diff
@@ -58,6 +54,67 @@
       notify:
       - restart omero-web
 
+    - name: OMERO.server self-signed certificate directory
+      become: yes
+      file:
+        path: /opt/omero/server/selfsigned
+        state: directory
+
+    - name: OMERO.server install openssl for generating certificates
+      become: yes
+      yum:
+        name: openssl
+        state: present
+
+    - name: OMERO.server self-signed certificate
+      become: yes
+      command: >-
+        openssl req
+        -new -nodes -x509
+        {{ omero_server_websocket_internal_cert_params }}
+        -keyout server.key
+        -out server.pem
+        -extensions v3_ca
+      args:
+        chdir: /opt/omero/server/selfsigned
+        creates: server.pem
+
+    # This self-signed cert is used to encrypt the websocket connection
+    # between Nginx and OMERO.server
+    - name: OMERO.server self-signed certificate pkcs12
```

Comment:
I'm guessing this isn't the kind of thing that `ome.ssl_certificate` can yet do for us.